kernel/SlippiFileWriter.c

```diff
@@ -177,16 +207,53 @@ static u32 SlippiHandlerThread(void *arg)
 
 	u32 writtenByteCount = 0;
 	driveTimer = read32(HW_TIMER);
+	driveTimerSet = false;
+
+	FATFS device;
+	bool failedToMount = false;
+	bool hasFile = false;
+	bool mounted = true;
+	const bool use_usb = ConfigGetUseUSB() != 1;
```

Comment:
`u32 UseUSB = ConfigGetUseUSB(); // Returns 0 for SD, 1 for USB`

based on the line above, shouldn't this be `const bool use_usb = ConfigGetUseUSB() (== 1);`?

Reply:
ConfigGetUseUSB refers to ISO device, so file write device is 'opposite'